pkgs/misc/emulators/punes/default.nix

```diff
@@ -0,0 +1,68 @@
+{ mkDerivation
+, stdenv
+, lib
+, fetchFromGitHub
+, unstableGitUpdater
+, qtbase
+, qtsvg
+, qttools
+, alsaLib
+, ffmpeg
+, libGLU
+, OpenGL
+, autoreconfHook
+, pkg-config
+, cmake
+}:
+
+mkDerivation rec {
+  pname = "punes";
+  version = "unstable-2021-04-25";
+
+  src = fetchFromGitHub {
+    owner = "punesemu";
+    repo = "puNES";
+    rev = "4b4c3495a56d3989544cb56079ce641da8aa9b35";
+    sha256 = "1wszvdgm38513v26p14k58shbkxn1qhkn8l0hsqi04vviicad59s";
+  };
+
+  postPatch = ''
+    substituteInPlace configure.ac \
+      --replace '`$PKG_CONFIG --variable=host_bins Qt5Core`/lrelease' '${qttools.dev}/bin/lrelease'
+  '';
+
+  nativeBuildInputs = [ qttools autoreconfHook pkg-config cmake ];
+
+  buildInputs = [ qtbase qtsvg ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [ alsaLib ffmpeg libGLU ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ OpenGL ];
+
+  enableParallelBuilding = true;
+
+  configureFlags = [
+    "--prefix=${placeholder "out"}"
+    "--without-opengl-nvidia-cg"
+    "--with-ffmpeg"
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    ./configure $configureFlags
+
+    runHook postConfigure
+  '';
```

Comment:
Done because the project uses mostly autoconf & only a single subproject uses cmake. Please suggest an appropriate `dontUseCmake = true`-esque option if there is one.

Reply:
Found it.